### PR TITLE
[#GEA] User can add new building

### PR DESCRIPTION
**Pull request by hadzija7**
Please approve

### CODE REVIEW

File: GraphicEditor/HelpClasses/Constants.cs [hunk 1]
```diff
@@ -22,6 +22,7 @@ public class Constants
         public const string PEDIATRICS = "pediatrics";
         public const string DERMATOLOGY = "dermatology";
         public const string ONCOLOGY = "oncology";
+        public const string EMPTYFIELD = "emptyField";
```
Comment (milicasiriski) [inline GraphicEditor/HelpClasses/Constants.cs]: EMTPY_FIELD

File: GraphicEditor/Resources/buildingStyles.txt [hunk 2]
```diff
@@ -0,0 +1 @@
+﻿
```
Comment (milicasiriski) [inline GraphicEditor/Resources/buildingStyles.txt]: Was this file left accidentally?

File: GraphicEditor/View/Windows/AddBuilding.xaml.cs [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+{
+    /// <summary>
+    /// Interaction logic for AddBuilding.xaml
+    /// </summary>
+    public partial class AddBuilding : Window
+    {
+        public AddBuilding()
+        {
+            this.DataContext = AddBuildingViewModel.GetInstance();
+            AddBuildingViewModel.GetInstance().Window = this;
```
Comment (milicasiriski) [inline GraphicEditor/View/Windows/AddBuilding.xaml.cs]: Avoid singletons

File: GraphicEditor/ViewModel/DBtest.cs [hunk 4]
```diff
@@ -0,0 +1,57 @@
...
+namespace GraphicEditor.ViewModel
+{
+    class DBtest
+    {
+        public static void DatabaseConnection()
+        {
+            for(int i = 0; i < 10; i ++)
+            {
+                Console.WriteLine("Cao");
+            }
```
Comment (milicasiriski) [inline GraphicEditor/ViewModel/DBtest.cs]: 😄

File: GraphicEditor/ViewModel/HospitalMapUserControlViewModel.cs [hunk 5]
```diff
@@ -1,18 +1,40 @@
...
         {
             _parent = parent;
             NavCommand = new MyICommand<string>(ChooseHospital);
+            AddCommand = new MyICommand<object>(AddBuilding);
+        }
+
+        private void AddBuilding(object button)
+        {
+            Button but = (Button)button;
+            System.Console.WriteLine(but.Content);
```
Comment (milicasiriski) [inline GraphicEditor/ViewModel/HospitalMapUserControlViewModel.cs]: probably left here by mistake

**Review by milicasiriski — CHANGES_REQUESTED** [review]

**Review by milicasiriski — APPROVED** [review]

**Review by lukaDoric — APPROVED** [review]

**Review by VladimirBudjen — APPROVED** [review]

**Review by markovidovic — APPROVED** [review]

**Pull request merged**